cli/src/params.rs

```diff
@@ -882,12 +883,24 @@ impl Assertions {
             let assert_output_facts: Option<Vec<InferenceFact>> = matches
                 .values_of("assert-output-fact")
                 .map(|vs| vs.map(|v| tensor::for_string(v).unwrap().1).collect());
+            let assert_op_count: Option<Vec<(String, usize)>> = sub
+                .values_of("assert-op-count")
+                .map(|vs| {
+                    vs.chunks(2)
+                        .into_iter()
+                        .map(|mut args| {
+                            Some((args.next()?.to_string(), args.next()?.parse().ok()?))
+                        })
+                        .collect()
+                })
+                .flatten();
```

Comment:
So how do we make this parser happy ? :) `--assert-op-count Add,2,Mul,4` ? 

We want an exemple in the help message (lines 323 and 269)